### PR TITLE
add super admin right

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1022
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/40

**Comment by itkg-alavieille** [comment]
can you update tests ?

**Comment by itkg-rboessel** [comment]
@itkg-alavieille done

File: BackofficeBundle/Security/Authorization/Voter/GroupSiteVoter.php [hunk 1]
```diff
@@ -76,6 +77,9 @@ public function vote(TokenInterface $token, $object, array $attributes)
                 }
             }
         }
+        if ($user instanceof UserInterface && $user->isSuperAdmin()) {
```
Comment (itkg-alavieille) [inline BackofficeBundle/Security/Authorization/Voter/GroupSiteVoter.php]: i think than you can do this check before the other  ``ìf (($user = $token->getUser()) instanceof GroupableInterface)``

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**